staffing/onboarding/vvv-setup.md

```diff
@@ -0,0 +1,89 @@
+# Vagrant Setup Instructions
+
+## Install VirtualBox, Vagrant, and VVV 
+
+- Start by following the [official installation guide for VVV on Github](https://github.com/varying-vagrant-vagrants/vvv/#installation---the-first-vagrant-up), but pay special attention to the notes below:
+ - We strongly recommend also installing all of the optional Vagrant plugins mentioned in step 4. These will only make it easier and more convenient for you to use VVV later.
+ - Between steps 6 and 7, you'll want to run the follwing commands to get the latest official release (without these you'll get the latest development build)
+
+```
+git fetch
+git checkout master
+```
+
+## Install VV
+
+Variable VVV (a.k.a. VV) provides some handy tools for building sites using VVV.
+
+- Follow the installation [instructions found on GitHub](https://github.com/bradp/vv#os-x-installation). 
+
+## Increase the Server Memory on Your Vagrant Box
+
+- Use the command line to navigate to the root directory of your Vagrant box (where your Vagrantfile is located).
+- Paste the following command into your terminal, hit Enter, and then type Control-D.
+
+```
+cat > Customfile
+config.vm.provider "virtualbox" do |v|
+  v.memory = 2048
+  v.cpus = 4
+end
+```
+
+- That created a new file, `Customfile` which increases the memory limit on your Vagrant box, as [recommended in the VVV documentation](https://github.com/Varying-Vagrant-Vagrants/VVV/wiki/Customising-your-Vagrant's-attributes-and-parameters).
+
+## Setup a Largo Blueprint for VV
+
+One useful feature of VV is that it allows you to create "blueprints" for site configurations. Below, we'll setup a basic blueprint for Largo sites.
```

Comment:
I don't believe so - the general `vv create` command will walk through site setup options, where you can specify a mysql database and external repo to use for `wp-content`, which I think is probably the way to go.

We'll need to modify our bitbucket repo a bit so things match up better, but that should be about it.